keyboards/keebio/levinson/keymaps/ksamborski/keymap.c

```diff
@@ -0,0 +1,212 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 3                                                                /* Symbols, Media */
+#define _RAISE 4                                                                /* Numbers, Arrows */
+#define _FUNC  5                                                                /* Function Keys */
+#define _NUMS  6                                                                /* Numpad */
+#define _ADJUST 16
+
+/* short layer aliases */
+#define _QW _QWERTY
+#define _LW _LOWER
+#define _RS _RAISE
+#define _NM _NUMS
+#define _AD _ADJUST
```

Comment:
You can replace all this with a single `enum` like so:

```
enum layers {
  _QW,
  _LW,
  _RS,
  _NM,
  _AD,
}
```

This ought to save you some firmware space too, as defining your last layer value as 16 is artificially increasing the number of layers created.

Reply:
Also, specifically, with the `_ADJUST` layer, having that at 16 means that layers 7-15 are created, but are empty, which artificially inflates the firmware size. 

And using a enum for the main layer defines makes things easier to manage, in the long run.